keps/693-multikueue/README.md

```diff
@@ -316,6 +323,71 @@ that ware created by this multikueue manager cluster and delete them if their lo
 - `WorkerLostTimeout` - defines the time a local workload's multikueue admission check state is kept Ready
 if the connection with its reserving worker cluster is lost.
 
+
+### MultiKueue Dispatcher API
+
+The MultiKueue Workload Controller follows a two-step process: first, it nominates worker clusters for each workload based on the configured dispatching mode;
+then, it synchronizes the workload by creating and managing its copies in the nominated clusters. 
+This ensures that only one cluster ultimately admits and runs the workload, while the management cluster maintains up-to-date status by tracking the admitted job.
+
+#### Cluster Nomination Phase
+Cluster nomination phase looks differently depending on the configured dispatching mode.
+MultiKueue supports three dispatching modes: AllClusters (`kueue.x-k8s.io/multikueue-dispatcher-all-at-once`),
+Incremental (`kueue.x-k8s.io/multikueue-dispatcher-incremental`), and External.
+
+* **AllClusters Mode**:  
```

Comment:
Let's align the naming consistently, either:
- AllAtOnce
- AllClusters
- AllWorkers
I'm good with any of them, maybe slightly leaning to AllAtOnce, but most importantly we should align them